### DISCUSSION TITLE
Crab and Darwinia collators

### DISCUSSION THREAD

**Discussion by SasoLithops** [darwinia-network/darwinia#1399] [General]
Hi all. Due to recent discussions in Telegram Channel and extremely high 80-90% fees from many Collators that make staking for nominators on Crab Network extremely unatractive it seems now is the time to act and do something about this.

Anyone having any idea how to change this so nominators receive stable rewards from point of fees being up to 10% please propose it here or in Telegram. I will later collect this ideas and put them to vote so we will choose the best one.

Or we can disscuss the best one and choose it directly. 
I leave that up to community to decide.

It would be great if developers and collators join this discussion. And express what can be done from their point of view. And what is reasonable fee from their perspective.

As always any opinion matters.

I will start:

My idea would be to keep the active set, but create lets say a Discord Channel where DAO members can act together and commonly  join a more respected collator with fees up to 10%, that don't get changed as often as they please. Since Laki suggested this joining a collator together first and gave for example Polkadotters, it would be great to hear from them if they will keep their fee in range up to 10% when they move in active set.

**Comment by Laki389** [comment]
> My idea would be to keep the active set, but create lets say a Discord Channel where DAO members can act together and commonly join a more respected collator with fees up to 10%, that don't get changed as often as they please. Since Laki suggested this joining a collator together first and gave for example Polkadotters, it would be great to hear from them if they will keep their fee in range up to 10% when they move in active set.

There is already a #staking channel on the Darwinia Discord. If there is enough activity, with 19M crab staking power someone from the Waiting pool could be picked into Active pool which will kick out into Waiting pool someone with 70-90% commission.

**Reply by SasoLithops** [reply]
Noted. Will add this to first idea when creating a voting proposal. 

Let's see what others have to say.

**Reply by Laki389** [reply]
haha, why should that be put to a vote, simple self-organization of crab chain delegates.

**Reply by SasoLithops** [reply]
Voting would be when more ideas are proposed and the DAO can decide on best one.

**Reply by Laki389** [reply]
I think this is the only idea/solution based on democracy and liberal economy.

Other possibilities are:

- Collators to be declared "enemies of the people", "greedy rich people" and under the threat of sanctions and violence, to decide with their own free will that they will have an infinite 1%-8%.

- Collators to take a monthly subsidy from the Darwinia Treasury to have low commissions (if it is determined that it is economically unviable and they will go into loss), but then the question arises why it was decided that 60% of the monthly inflation will go to the Treasury to now again goes back to the Collators, this would be an admission that it was wrong to do so.

- Darwinia developers to code $crab to pump x10 or x100  :D which will allow collators economic viability with low commissions :D

**Reply by SasoLithops** [reply]
Well hopefully there are smarter people than us and another idea can be proposed😉

Just to clarify this proposal is in no way meant to choose a way to harm collators, but search for better solution.

I really hope some of collators explain their expenses so we find solution that suits us all. That stakers and collators earn its fair percentage.

**Comment by SasoLithops** [comment]
I am adding Laki's calculations to make more sense to discussion:

Laki:
i have done some math. 146 000 Crab "pass" through collators daily. There 14 collators, so 146k/14= 10 428 Crab.   At Snowswap 10 428 Crab can be swapped for 904 Ring and 904 Ring worths 2.117168 $.  1 month would be 30*2.117168 = less then 64$, if they have 100% commission.       10% commission gives 6.4$ month, 20% gives 12.8$, etc..

i did this math at Darwinia Unofficial Community Chat... if a crab collator have 100% commission  will "dig" less then 64$ in a month. If the costs are $60, there should be almost a 94% commission to break even 🙈😬

**Comment by hackfisher** [comment]
Fixed by https://github.com/darwinia-network/DIPs/blob/main/DIPs/dip-1.md
